frontend/src/pages/interview.js

```diff
@@ -0,0 +1,172 @@
+import { Component } from 'react'
+import { Button, Form, FormGroup, Label, Input, Container } from 'reactstrap'
+import Head from '../components/head'
+import Nav from '../components/nav'
+import Link from 'next/link'
+
+type Props = {}
+class Interview extends Component<Props> {
+  constructor(props) {
+    super(props)
+    this.state = { value: '' }
+    this.handleSubmit = this.handleSubmit.bind(this)
+  }
+  handleSubmit(event) {
```

Comment:
nit: fat arrows?? hehe

Reply:
oh shoot is that function being used?
let me check